hwc_display.ini

```diff
@@ -25,6 +26,11 @@ LOGICAL_DISPLAY="0:3"
 # sub-display-index: start from 0, should be the available displays number, follow the order of the connected logical displays
 MOSAIC_DISPLAY="0+1+2"
 
+# Float display, with format "sub-display-index:left+top+right+bottom"
+# sub-display-index: start from 0, should be the available displays number, follow the order of the connected logical displays
```

Comment:
We should add that this becomes the resolution of the display irrespective of the modeset ?

Reply:
Didn't get what did you mean by irrespective of modeset. If FLOAT=true then only it becomes the resolution. Esample, if LOGICAL=true then it becomes on top of it. (future)

Reply:
I mean ir-respective of display resolution being 1920x1080, this becomes the resolution of display in this case

Reply:
Elaborated the description